user-manual/access-content/context-menu.rst

```diff
@@ -437,9 +437,10 @@ elements:
 Subject and Place browse pages
 ==============================
 
-:term:`Subject` and :term:`Place` terms can be used in AtoM as
+:term:`Subject` and :term:`Places` terms can be used in AtoM as
 :term:`access points <access point>`, and then browsed to explore the
-:term:`archival descriptions <archival description>` to which they are linked.
+:term:`archival descriptions <archival description>` and/or 
+:term:`authority records <authority record>` to which they are linked.
 For more information, see:
 :ref:`Browse subjects and places <browse-subjects-places>`.
```

Comment:
Note that for user-manual/access-content/images/place-context-menu.png which appears right after this text (can't comment on actual binary files), the open https://projects.artefactual.com/issues/13332 issue also applies.

Reply:
Same goes for user-manual/access-content/images/browse-place-term2.png
user-manual/access-content/images/term-view-page.png